src/wallet/scriptpubkeyman.cpp

```diff
@@ -304,6 +344,19 @@ void LegacyScriptPubKeyMan::MarkUnusedAddresses(const CScript& script)
                 WalletLogPrintf("%s: Topping up keypool failed (locked wallet)\n", __func__);
             }
         }
+
+        auto it = mapKeyMetadata.find(keyid);
```

Comment:
f5eaa90 nit: I reckon this new code section could use a comment.

Reply:
Done